src/com/richard/walker/weatherstation/constants/Cities.java

```diff
@@ -0,0 +1,32 @@
+package com.richard.walker.weatherstation.constants;
+
+public final class Cities {
+    public final static String[] EAST_COAST_CITIES = new String[]{
+            "Raleigh",
+            "Durham",
+            "Chapel Hill",
+            "Charlotte",
+            "Atlanta",
+            "Miami",
+            "Orlando",
+            "New York",
+            "Boston",
+            "Washington D.C."
+    };
+
+    public final static String[] WEST_COAST_CITIES = new String[] {
+            "San Francisco",
+            "Portland",
+            "Seattle",
+            "Los Angeles",
+            "San Diego",
+            "Oakland",
+            "Salem",
+            "Olympia",
+            "Spokane"
+    };
+
+    public Cities() {
```

Comment:
Not sure what you're trying to do here. If we want to prevent instantiation, we can privatize the constructor.